engines/bastion_katello/app/assets/javascripts/bastion_katello/common/csv-export-url.service.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @ngdoc service
+ * @name  Bastion.common.service:CsvExportUrl
+ *
+ * @description
+ *   Helper service that contains functionality common amongst csv urls
+ */
+angular.module('Bastion.common').service('CsvExportUrl', ['$location', '$httpParamSerializer',
+    function ($location, $httpParamSerializer) {
+        this.refreshCsvLink = function(params) {
```

Comment:
Oh..so that link generation depends on the search params, every time the user enters a different search query, we are generating a new URL. So refreshCsvLink came to my mind..Do you prefer getCsvLink?